wca-regulations.md

```diff
@@ -405,7 +405,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
     - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).
     - A6e) After releasing the puzzle, the competitor must not touch or move the puzzle until the judge has inspected the puzzle. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
-    - A6f) The competitor must not reset the timer until the judge has recorded the result on the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
+    - A6f) The competitor must not reset the timer until both the competitor and the judge has signed the result on the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```

Comment:
A7c4 mentions "the judge has recorded and signed the attempt". So maybe replace "has signed the result on the score sheet" to "has signed the attempt" for consistency?

Reply:
```suggestion
    - A6f) The competitor must not reset the timer until both the competitor and the judge has signed the attempt. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```